src/samples/audio/detect_test.c

```diff
@@ -85,19 +91,23 @@ struct comp_data {
 	uint32_t keyphrase_samples; /**< keyphrase length in samples */
 	uint32_t drain_req; /** defines draining size in bytes. */
 	uint16_t sample_valid_bytes;
-	struct kpb_event_data event_data;
 	struct kpb_client client_data;
 
 #if CONFIG_KWD_NN_SAMPLE_KEYPHRASE
 	int16_t *input;
 	size_t input_size;
 #endif
-
 	struct ipc_msg *msg;	/**< host notification */
 
 	void (*detect_func)(struct comp_dev *dev,
 			    const struct audio_stream __sparse_cache *source, uint32_t frames);
 	struct sof_ipc_comp_event event;
+
+#if CONFIG_AMS
+	uint32_t kpd_uuid_id;
+#else
+	struct kpb_event_data event_data;
```

Comment:
ideally small code moves would be done in a small separate patch.

Reply:
+1, git commits move around.

https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes